docs/CN/RMSNorm/README.md

```diff
@@ -0,0 +1,33 @@
+# RMSNorm
+
+**RMSNorm** (Root Mean Square Layer Normalization) 是一种针对 Layer Norm 改进的归一化方法。
+
+## What is Layer Norm?
+
+Layer Norm 是 NLP 任务中一种常见的归一化方法，旨在解决深度神经网络中存在的 Internal Covariate Shift 问题，即层与层间的输入分布在训练过程中会不断发生变化。这种现象可能会导致网络训练速度减慢，难以收敛等问题。Layer Norm 在每个样本上统计所有维度的值，计算均值和方差进行归一化操作，使数据在训练过程保持相同的分布，从而减轻 Internal Covariate Shift 问题。
+
+Layer Norm 的计算过程可以公式化表示如下：
+
+$$
+\bar{a}_i = \cfrac{a_i -\mu}{\sigma}g_i\\ \mu = \cfrac{1}{n} \sum_{i=1}^{n}a_i\qquad \sigma=\sqrt{\cfrac{1}{n}\sum_{i=1}^{n}(a_i - \mu)^2}
+$$
+
+其中，$a\in\mathbb{R}^n$ 表示 Layer Norm 进行归一化的向量， $\bar{a}\in\mathbb{R}^n$ 表示经过 Layer Norm 规范后的向量。 $\mu$ 和 $\sigma$ 分别表示输入向量 $a$ 的均值和方差。$g\in\mathbb{R}^n$ 是一组初始化为 $1$ 的增益系数，用来缩放归一化后的结果，来保证归一化操作不会破坏输入的信息。
```

Comment:
May need one extra space for math equation rendering.

```suggestion
其中，$a\in\mathbb{R}^n$ 表示 Layer Norm 进行归一化的向量， $\bar{a}\in\mathbb{R}^n$ 表示经过 Layer Norm 规范后的向量。 $\mu$ 和 $\sigma$ 分别表示输入向量 $a$ 的均值和方差。 $g\in\mathbb{R}^n$ 是一组初始化为 $1$ 的增益系数，用来缩放归一化后的结果，来保证归一化操作不会破坏输入的信息。
```

Reply:
Done.